src/Modules/WorldRecordModule/Models/WorldRecord.cs

```diff
@@ -0,0 +1,15 @@
+﻿using EvoSC.Common.Util;
+
+namespace EvoSC.Modules.Official.WorldRecordModule.Models;
+
+public class WorldRecord
+{
+    public required string Name { get; init; }
+    public required int Time { get; init; }
```

Comment:
Use the `RaceTime` class from Common.Utils instead of the int. Gives greater control and compatability, returns the correct format with just .ToString()